packages/grub-btrfs/grub-btrfs.pacscript

```diff
@@ -0,0 +1,21 @@
+pkgname=grub-btrfs
+pkgver=4.13
+pkgdesc='Include btrfs snapshots in GRUB boot options'
+url='https://github.com/Antynea/grub-btrfs'
+license=('GPL-3.0-only')
+depends=('btrfs-progs' 'grub2-common')
+optdepends=(
+  'snapper: For snapper support'
+  'inotify-tools: For grub-btrfsd daemon'
+)
+backup=('etc/default/grub-btrfs/config')
```

Comment:
anything in /etc automatically is considered a config file, so this like is redundant.